tests/e2e/normalize.go

```diff
@@ -562,7 +579,8 @@ func normalizeHTTPResponses(t *testing.T, events test.LogEntries) {
 		visitor.replacePaths[".job.startTime"] = "2024-04-01T12:34:56.123456Z"
 		visitor.replacePaths[".job.createTime"] = "2024-04-01T12:34:56.123456Z"
 		visitor.replacePaths[".currentStateTime"] = "2024-04-01T12:34:56.123456Z"
-		visitor.sortSlices.Insert(".environment.experiments")
+		visitor.sortAndDeduplicateSlices.Insert(".environment.experiments")
+		visitor.sortAndDeduplicateSlices.Insert(".environment.sdkPipelineOptions.options.experiments")
```

Comment:
Can we do sorting in the MockGCP side?

Reply:
We also need to sort the GCP output though, it currently is randomly ordered I believe